bazel-cache/templates/deployment.yaml

```diff
@@ -26,6 +26,21 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - -dir
+            - "{{ .Values.dir }}"
+            - -max_size
+            - "{{ .Values.max_size }}"
+            - -port
+            - "{{ .Values.port }}"
+          {{- range $key, $value := .Values.extraArgs }}
+            {{- if $value }}
+            - -{{ $key }}
+            - "{{ $value }}"
+            {{- else }}
+            - -{{ $key }}
+            {{- end }}
+          {{- end }}
```

Comment:
I'm not sure this is the neatest or easiest way to do it. Maybe just expect an array of strings, not a dict, and do:
```
{{- if .Values.extraArgs }}
{{ toYaml .Values.extraArgs | indent 10 }}
{{- end }}
```
(from https://github.com/kubernetes/charts/blob/master/stable/kubernetes-dashboard/templates/deployment.yaml#L35 - check the indent as well)